spec/truffle/methods/String.txt

```diff
@@ -31,6 +31,7 @@ codepoints
 concat
 count
 crypt
+dedup
```

Comment:
FYI we shouldn't modify these files, they should reflect the MRI version we import.
But it's no big deal, let's leave as-is for this PR, it will overridden on 3.2 import.

Reply:
What is the right way to do this? Without this tests will complain that the method is not exposed?

Reply:
See https://github.com/oracle/truffleruby/blob/master/spec/truffle/methods_spec.rb, so you'd just use `jt tag` to mark as extra method, until the 3.2 import which would then regenerate that.